orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -359,6 +359,29 @@ void IterateSubgraphFromNode(Graph& graph,
 }
 }  // namespace
 
+void RemovePrintDensityFlag(Graph& graph, bool& modified, const logging::Logger& logger) {
+  GraphViewer graph_viewer(graph);
+  const auto& node_topology_list = graph_viewer.GetNodesInTopologicalOrder();
```

Comment:
can we pass node_topology_list as argument. graph_viewer constrution woul take some overhead.

Reply:
fixed, thx.